src/ripple/protocol/Feature.h

```diff
@@ -34,6 +34,52 @@ uint256
 feature (const char* name);
 /** @} */
 
+constexpr size_t FeatureBitsetSize = 32;
```

Comment:
My personal suspicion is that our features will outgrow 32 bits not very far in the future.  We've had `Feature`s for about 1 1/2 years and there are already 18 of them.  Given that, I suggest we turn the `assert` that guards the `FeatureToIndex` map construction into a `LogicError`.